src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api.Tests/Services/ContentApiClientMock.cs

```diff
@@ -11,6 +11,11 @@ internal class ContentApiClientMock : IContentApiClient
 {
     private readonly DataFixture _dataFixture = new();
 
+    public Task<Either<ActionResult, PublicationCacheViewModel>> GetPublication(Guid publicationId)
+    {
+        throw new NotImplementedException();
```

Comment:
Still going to do a PR to remove the `ContentApiClientMock`?

Reply:
Ah yes! I forgot. I was going to do it as part of this PR

Reply:
Removed! :)